common/scripted_triggers/iisc_triggers.txt

```diff
@@ -98,12 +98,34 @@ iisc_is_scientist_idle = {
 	}
 }
 
+# this = corp
+# hq set
+iisc_can_spawn_ships = {
+	event_target:iisc_hq_planet = {
+		has_orbital_bombardment = no
+		# starbase is not captured
+		solar_system.starbase.fleet.owner = {
+			# the fleet owner is the controller
+			prev.solar_system.starbase.owner = {
+				# the starbase owner is the owner
```

Comment:
the starbase owner is the _owner_? is this not a tautology? this seems redundant - do you not mean to have `.controller` at the end of this scope change?